app.rb

```diff
@@ -1,9 +1,32 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/peep.rb'
 
 class Chitter < Sinatra::Base
+
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    'Peeps!'
+  end
+
+  get '/peeps' do
+    @peeps = Peep.all
+    erb :'peeps/index'
+  end
+
+  get '/peeps/new' do
+    erb :'peeps/new'
+  end
+
+  post '/peeps' do
+    @peep_date = Time.new.strftime("%Y-%d-%m")
```

Comment:
I like this way of entering the time to the DB. You could potentially also make the DB enter the time/date automatically when you create a new row (peep)